translations-desktop/handleDesktopTranslations.sh

```diff
@@ -55,17 +55,23 @@ do
     crashreporter=""
   fi
 
-  lupdate src/gui/ src/cmd/ src/common/ $crashreporter src/csync/ src/libsync/ $resources -ts /branches/$version.ts
+  # copy existing translation file from the branch, this allows `lupdate` to e.g. keep existing plural forms
+  cp "translations/client_en.ts" "/branches/$version.ts"
+
+  lupdate -no-obsolete src/gui/ src/cmd/ src/common/ $crashreporter src/csync/ src/libsync/ $resources -ts /branches/$version.ts
 done
 
 # Merge source translation files and filter duplicates
+# This will be used as the source file for Transifex, it contains all translatable strings from each branch
 lconvert -i /branches/*.ts -o /merged_en.ts
 
-# Fix missing <numerusform> elements (always two are required but lconvert strips out one)
```

Comment:
the `<numerusform>` workaround doesn't seem to be required anymore with the more recent Qt version in Ubuntu 24.04